package.json

```diff
@@ -3,7 +3,7 @@
 	"version": "26.1.0",
 	"private": true,
 	"scripts": {
-		"start": "node ./bin/www",
```

Comment:
this will produce `      message: "should have required property 'COOKIE_SECRET'"` when start without the required setting, we must ensure that all settings from development are defined in production...